modules/dnn/test/test_caffe_importer.cpp

```diff
@@ -276,16 +361,19 @@ TEST_P(Reproducibility_SqueezeNet_v1_1, Accuracy)
 }
 INSTANTIATE_TEST_CASE_P(/**/, Reproducibility_SqueezeNet_v1_1, availableDnnTargets());
 
-TEST(Reproducibility_AlexNet_fp16, Accuracy)
```

Comment:
This test is used to check an accuracy of network with FP16 weights received from `shrinkCaffeModel` but FP32 computations. I think we can avoid models tests with `DNN_TARGET_OPENCL_FP16` target and compare it with default backend and CPU target at `test_backends`.

Reply:
ok,  Then i will do googlenet and alexnet fp16 test in test_backends, and leave this test unchanged here.